tests/cross_fw/test_templates/template_test_nncf_tensor.py

```diff
@@ -112,7 +112,8 @@ def test_operators_tensor(self, op_name):
         assert res.dtype == res_nncf.data.dtype
         assert all(res == res_nncf.data)
         assert isinstance(res_nncf, Tensor)
-        assert res_nncf.device == nncf_tensor_a.device
+        if not (self.backend() == TensorBackend.tf and self.device() == TensorDeviceType.CPU):
```

Comment:
Could you briefly explain the reason for these changes while you return specific device tensors for TF?

Reply:
the result of binary operation with two CPU TF tensors is GPU TF tensor by default. it means that the following check would fail on CPU.